naming/README.md

```diff
@@ -0,0 +1,76 @@
+# coreboot, LinuxBoot, NERF... What?
+
+Naming is hard. You may have heard of `coreboot`, `NERF`, `u-root`, `systemboot`, etc. If you are confused, well, you're not alone.
+
+# LinuxBIOS
+`LinuxBIOS` is a project originated in 1999 from Ron Minnich, Stefan Reinauer and others. It is not much younger than UEFI, but they were already experimenting the idea of running Linux as firmware! Like many great ideas, it was way ahead of its time. At that time Linux was not mature enough for hardware initialization project, and while LinuxBIOS was successful in several performance-and-reliability critical environments, it didn't see mass adoption.
+
+# coreboot
+LinuxBIOS became `coreboot` in 2008. It is effectively the same project that evolved over time. `coreboot` (spelled lowercase) is a complete open source system firmware package, aimed at replacing proprietary implementations. It's also one of the most mature and well-maintained open source firmware projects.
+
+`coreboot` supports a **wide varieties of platforms**, and has a **modular architecture**. It provides **platform initialization** (CPU, DRAM, PCI, ACPI, SMBIOS, etc), **a filesystem** (CBFS) suitable for on-firmware storage, integration with **vendor extensions and blobs**, a friendly license, and a **wide variety of payloads**.
+
+This modular design also **enables various bootloader scenarios**, through coreboot payloads. Among the various options, there are:
+* *SeaBIOS*, a very popular open source BIOS implementation
+* *depthcharge*, the blazing-fast boot payload for ChromeOS/Chromebooks
+* *LinuxBoot*, of course! More details below
+* *UEFI*, via the open-source EDK II.
+
+# LinuxBoot
+LinuxBoot is not a product, but rather a **concept**. It's the idea of **booting Linux (OS) with Linux (system firmware)**. In a way, the same concept pioneered by LinuxBIOS.
+
+"LinuxBoot" is also often used as an umbrella name at Facebook to indicate how we do open source firmware, i.e. coreboot + Linux + u-root + systemboot. **Imagine it like a Linux distribution, but for firmware**. It is a collection of various open source components, glued together to work as a consistent firmware OS.
```

Comment:
`at Facebook` to indicate how `we` do
I know you are from Facebook so I understand the `we` but the book should probably be authored from the linuxboot community point of view. So `they` might be more accurate.

Reply:
Thanks for catching this. This comes originally from an internal post that I made a few months ago, and I tried to remove all the references to "us" and "me" but some of them slipped through :)